src/shared/components/query/QueryContainer.tsx

```diff
@@ -65,6 +66,9 @@ export default class QueryContainer extends React.Component<QueryContainerProps,
                 {
                     <OverlappingStudiesWarning studies={this.store.selectedStudies}/>
                 }
+                {
+                    <UnknownStudiesWarning ids={this.store.unknownStudyIds} />

```

Comment:
make sure that if both of these errors are showing, layout looks ok. please put screenshot in PR

Reply:
@alisman @kalletlak 
![screen shot 2017-12-07 at 1 58 08 pm](https://user-images.githubusercontent.com/1334004/33733129-b88c53ce-db56-11e7-829b-2b0b9c60d0cc.png)

Reply:
FYI we can't show the full study name, because access to those study ids are forbidden